NuGet.config

```diff
@@ -1,8 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <packageSources>
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
-    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
-    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-</configuration>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />

```

Comment:
We're currently on CLI version 2.0.0-preview1-005899 and M.NETCore.App https://dotnet.myget.org/feed/aspnetcore-ci-dev/package/nuget/Microsoft.NETCore.App/2.0.0-preview1-002101-00. It might be the SDK that's bumping up the M.NETCore.App version to 2.0.0-preview1-002106-00. Does using the slightly older SDK work for now? Regardless, @kichalla should be able to tell when we're meant to update to a newer build of the SDK which should possibly fix this issue